### DISCUSSION TITLE
[Bug]: how do i open it?

### DISCUSSION THREAD

**Discussion by AverageVirusTotalEnjoyer** [winstxnhdw/lc-hax#57] [Q&A]
### What happened?

ewe

### Current Commit Hash

d

### Injector

- [X] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have **recursively** cloned the repository as instructed in the [README](https://github.com/winstxnhdw/lc-hax/edit/main/README.md)
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment]
Open what?

**Comment by AverageVirusTotalEnjoyer** [comment]
nvm im genuinely stupid
